schema.json

```diff
@@ -907,6 +907,12 @@
         "force_reload": true,
         "id": "homepage_show_carousel"
       },
+      {
+        "type": "checkbox",
+        "label": "Show Carousel Arrows",
```

Comment:
Should have the capitalization changed to be `Show carousel arrows`, but looks good other than that.